### PR TITLE
Update search-the-audit-log-in-security-and-compliance.md for adding Forms Collection auditing logs

### PR DESCRIPTION
**Pull request by JessieChenOutlook**
Adding the Forms Collection related auditing logs, including: CollectionCreated,  CollectionUpdated,   CollectionHardDeleted,   CollectionSoftDeleted,  CollectionRenamed,  MovedFormIntoCollection,  MovedFormOutofCollection

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md [hunk 1]
```diff
@@ -965,6 +965,13 @@ If a Forms activity is performed by a coauthor or an anonymous responder, it wil
 |Enabled people in my organization collaboration<sup>*</sup>|EnableSameOrgCollaboration|Form owner turns on the setting allowing users in the current organization to view and edit the form.|
 |Enabled specific people collaboration<sup>*</sup>|EnableSpecificCollaboaration|Form owner turns on the setting allowing only specific people or specific groups in the current organization to view and edit the form.|
 |Connected to Excel workbook<sup>*</sup>|ConnectToExcelWorkbook|Connected the form to an Excel workbook. <br><br>Property ExcelWorkbookLink:string indicates the associated Excel workbook ID of the current form.|
+|Created a collection|CollectionCreated|Form owner created a collection.|
+|Updated a collection|CollectionUpdated|Form owner updated a collection property.|
+|Deleted collection from the Recycle Bin|CollectionHardDeleted|Form owner hard deleted a collection from Recycle Bin.|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md]:
```suggestion
|Deleted collection from Recycle Bin|CollectionHardDeleted|Form owner hard deleted a collection from Recycle Bin.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JessieChenOutlook, kindly commit the minor suggested change. Many thanks.

**Comment by markjjo** [comment]
@JessieChenOutlook PR merged. Thanks for adding these new Forms activities.

**Pull request merged**